src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -171,10 +171,16 @@ protected function setCustomModes(PDO $connection, array $config)
     /**
      * Get the query to enable strict mode.
      *
+     * @param  int|null $version
+     *
      * @return string
      */
-    protected function strictMode()
+    protected function strictMode($version)
     {
+        if ($version === 8) {
```

Comment:
`$version > 7` is probably safest, otherwise people are gonna get confused

Reply:
Possible they will put `7.4` `8.0` or whatever into the config, and if we update to `>`, that'll still work. We could also take a string, and do a version_compare.

Reply:
I updated the commit to check against the PDO::ATTR_SERVER_VERSION attribute and the code now uses the > operator as you suggested.